pkg/asset/quota/quota.go

```diff
@@ -81,7 +81,8 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 		}
 		q, err := quotaaws.Load(context.TODO(), session, ic.AWS.Region, services...)
 		if quotaaws.IsUnauthorized(err) {
-			logrus.Warnf("Missing permissions to fetch Quotas and therefore will skip checking them: %v, make sure you have `servicequotas:ListAWSDefaultServiceQuotas` permission available to the user.", err)
+			logrus.Debugf("Missing permissions to fetch Quotas and therefore will skip checking them: %v, make sure you have `servicequotas:ListAWSDefaultServiceQuotas` permission available to the user.", err)
```

Comment:
Should also be a Info message? I am not sure if the Debug msgs are always visible (should a debug flag be present ?)

Reply:
I'll let @staebler weigh in on whether `Infof` is preferable over `Debugf`

Reply:
I think it is appropriate as is. The user gets an INFO message that the quota checks are skipped. There is additional detail in a DEBUG message, which is always written to .openshift_install.log.